DXMainClient/DXGUI/Multiplayer/GameLobby/SkirmishLobby.cs

```diff
@@ -60,6 +61,11 @@ public override void Initialize()
             ddPlayerSides[0].SelectedIndexChanged += PlayerSideChanged;
         }
 
+        protected override void AddNotice(string message, Color color)
+        {
+            // do nothing
```

Comment:
Could use `XNAMessageBox.Show` to display the notice if it'd result in a better user experience.

Reply:
Works for me. Will add.